server/chat-plugins/mafia-data.js

```diff
@@ -469,18 +492,31 @@ const roles = {
 		id: `doityourselfer`,
 		memo: [`Do-It-Yourselfer: Each Night, you may use a One-Shot ability of a discarded role. You may not use a discarded role multiple times.`],
 	},
+	dragoon: {
+		name: `Dragoon`,
+		id: `dragoon`,
+		memo: [`Dragoon:  If you are one of the last 3 players alive, you instantly win. Has no factional kill but says one message anonymously each night through the host. The message takes place at the same time as the werewolves’ kill, if one exists and is compulsive.
```

Comment:
I'd rephrase the first sentence as `You win if you are one of the last 3 players alive.` for consistency.